packages/component-library/tsconfig.dist.json

```diff
@@ -3,6 +3,7 @@
   "include": ["components/**/*", "draft/**/*", "util/**/*", "index.ts"],
   "compilerOptions": {
     "noEmit": false,
+    "allowJs": false,
```

Comment:
Make an issue to make this default at the repo level (will require some conversion of JS to TS in the site)

Reply:
Issue to resolve: https://github.com/cultureamp/kaizen-design-system/issues/83